ci/azure-pipelines-nightly.yml

```diff
@@ -78,6 +80,7 @@ jobs:
   - template: ../ci/install-package-dependencies.yml
   - bash: |
       set -x
+      pwd
```

Comment:
Is this needed, if so why?

Reply:
Leftover from development, I will remove that